subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -790,7 +791,8 @@ static void isr_rx(void *param)
 		radio_tmr_start_us(0, start_us);
 
 		/* Add 4 us + 4 us, as radio was setup to listen 4 us early */
```

Comment:
I suggest to align this comment to code below. In other case people may start to thing that there is no need to update comments if code is changed.